packages/react-native-editor/src/api-fetch-setup.js

```diff
@@ -10,15 +10,21 @@ const SUPPORTED_ENDPOINTS = [
 	/wp\/v2\/search\?.*/i,
 ];
 
+// [ONLY ON ANDROID] The requests made to these endpoints won't be cached.
+const DISABLED_CACHING_ENDPOINTS = [ /wp\/v2\/(blocks)\/?\d*?.*/i ];
+
 const setTimeoutPromise = ( delay ) =>
 	new Promise( ( resolve ) => setTimeout( resolve, delay ) );
 
+const shouldEnableCaching = ( path ) =>
+	! DISABLED_CACHING_ENDPOINTS.some( ( pattern ) => pattern.test( path ) );
+
```

Comment:
Prior to these changes, the requests were always cached ([related code](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/f32b7cac415478ccc7fb9f460e1a062f2435fb90/fluxc/src/main/java/org/wordpress/android/fluxc/store/ReactNativeStore.kt#L60)), so by default requests will be cached unless they're explicitly added to the `DISABLED_CACHING_ENDPOINTS` pattern.

Reply:
Makes sense. Thanks for the context @fluiddot I appreciate it. I was thinking that since `isPathSupported` is included in the [`api-fetch-setup.test.js`](https://github.com/WordPress/gutenberg/blob/f549c3245b8de3e2c5636f4e5e41900e101a5797/packages/react-native-editor/src/test/api-fetch-setup.test.js) file we should probably include `shouldEnableCaching ` as well? Wdyt?

Reply:
Also for readability purposes, I was wondering what you think about putting the `shouldEnableCaching` function below `isPathSupported `? IMO, it would allow the reader to quickly notice that both functions are sharing the same pattern matching constructs.

Reply:
I added a unit test for `shouldEnableCaching` as well as move the function in this [commit](https://github.com/WordPress/gutenberg/pull/31186/commits/718226095f5e4d399016841138667df03d40cc0a).